File: cms-sm/src/main/java/com/sophia/cms/sm/model/DataFilterModel.java
package com.sophia.cms.sm.model;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

/**
 * Created by lenovo on 2017/8/13.
 */
@Data
public class DataFilterModel {

    @ApiModelProperty(value = "标题")
    private String title;

    @ApiModelProperty(value = "字段名")
    private String field;

    @ApiModelProperty(value = "字段类型")
    private String fieldType;

//    @ApiModelProperty(value = "数据类型")
//    private String dataType;

    @ApiModelProperty(value = "表达式")
    private String expression;

    @ApiModelProperty(value = "索引")
    private String sort;

    @ApiModelProperty(value = "扩展")
    private String extra;
}

File: client/src/app/sm/model/datafilter.model.ts
import { FieldModel } from './field.model';

export class DataFilterModel {
  constructor() {
    this.title = '';
    this.field = '';
    this.fieldType = '';
    this.expression = '';
    this.sort = '';
    this.extra = '';
  }
  // 标题
  title: string;

  // 字段名
  field: string;

  // 字段类型
  fieldType: string;

  // 表达式
  expression: string;

  // 索引
  sort: string;

  // 扩展
  extra: string;
}
